design-documents/graph-ql/coverage/search.md

```diff
@@ -0,0 +1,88 @@
+# Queries
+
+```graphql
+type Query {
+    multiSearch(phrase: String!, productSize: Int = 10): MultiSearchResponse!
```

Comment:
Can an example be provided where `multiSearch` is used, where `productSearch` couldn't have been used instead?

I'm still not quite clear on what the difference are.

Reply:
removed